app/notification.go

```diff
@@ -465,8 +533,19 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 		}
 	}
 
+	usersToReceiveNotification := make(model.StringArray, 0)
+
 	if len(mentionedUsersList) != 0 {
-		message.Add("mentions", model.ArrayToJson(mentionedUsersList))
+		usersToReceiveNotification = append(usersToReceiveNotification, mentionedUsersList...)
+	}
+
+	if len(notificationsForCRT.Desktop) != 0 {
+		usersToReceiveNotification = append(usersToReceiveNotification, notificationsForCRT.Desktop...)
+	}
+
+	if len(usersToReceiveNotification) != 0 {
+		usersToReceiveNotification = model.RemoveDuplicateStrings(usersToReceiveNotification)
+		message.Add("mentions", model.ArrayToJson(usersToReceiveNotification))
```

Comment:
I'm not sure about this side-effects of this change. `usersToReceiveNotification` is mentionedUsers + followers. Adding followers to the `mentions` field of the websocket message doesn't seem correct.

Reply:
This change was done so that we won't have to call the thread in the front-end to see if the user is following or not.
But we could just add another field called `followers` so that we don't mix.